apps/daimo-mobile/src/view/screen/link/NoteScreen.tsx

```diff
@@ -216,10 +221,11 @@ function NoteDisplayInner({
   const netDollarsReceivedStr = getAmountText({ dollars: netRecv });
 
   // On success, go home, show newly created transaction
-  const goHome = useExitToHome();
+  const backHome = useExitBack();
```

Comment:
`goBack`? this won't necessarily go all the way to the home screen.

Reply:
Since `NoteScreen` is in HomeTab now I believe it'll always go to Home Screen, but edited name nonetheless.